openslides/motions/signals.py

```diff
@@ -68,7 +68,7 @@ def setup_motion_config(sender, **kwargs):
         subgroup=ugettext_lazy('General'))
 
     # Amendments
-
+    # Amendments currently not implemented. (TODO: Implement it like in OpenSlides 1.7.)
```

Comment:
If we do not have amendments we might disable these two config variables, don't we?